apps/accounts/verify.py

```diff
@@ -0,0 +1,35 @@
+# Don't import any django stuff here.
+
+import requests
+
+
+def fxa_identify(code, config=None):
+    """Get an FxA profile for an access token. If idenficiation fails return
+    an empty dict."""
+    token = get_fxa_token(code, config).get('access_token')
+    if token:
+        return get_fxa_profile(token, config)
+    else:
+        return {}
+
+
+def get_fxa_token(code, config):
+    response = requests.post(config['oauth_uri'] + '/token', data={
+        'code': code,
+        'client_id': config['client_id'],
+        'client_secret': config['client_secret'],
+    })
+    if response.status_code == 200:
+        return response.json()
+    else:
+        return {}
```

Comment:
additionally, we should get some information on 500 responses, at least in logging. A 500 response should not be catchable because that means there is an upstream error in the API so it cannot be trusted.